### DISCUSSION TITLE
not able to received test mail in inbucket

### DISCUSSION THREAD

**Discussion by techsachin1** [inbucket/inbucket#365] [Q&A]
hi i have install inbucket on my VM. do i need domain name to test it . i didnt received any mail. i have change port 2500 to 25. i have outlook n gmail on VM , i have tried from same VM and another PC mail. but not received any single mail. kindly share what re-requisite to test mail

**Comment by jhillyerd** [comment]
If you want inbucket to capture email for a single domain, you will need to setup a DNS MX record for that domain (that points to inbucket) that is visible to the machines trying to send mail.  If you control your own local DNS servers, you may not need to buy a real domain for this, but if you are using public DNS servers you would need to have a real domain.

Alternatively, if you want Inbucket to capture all mail from a particular server for testing purposes, you typically configure your application to use your inbucket as its SMTP host.  Or configure the machines local email server to consider your inbucket a smart relay.

**Reply by techsachin1** [reply]
can you please guide how to  setup a DNS MX record for that domain or  local email server to consider your inbucket a smart relay.
